### ISSUE TITLE
G-error is always 0

### ISSUE THREAD

**Issue by dotunadegbite** [carpedm20/DCGAN-tensorflow#246]
I am using my own custom dataset and my g_error is always at 0 which I find to be odd. Is this okay? If not do I need to add labels and incorrect examples to my training set?
